lib/attache/resize_job.rb

```diff
@@ -52,7 +52,8 @@ def make_safe_filename(str)
     end
 
     def thumbnail_for(closed_file:, target_geometry_string:, extension:, max: 2048)
-      thumbnail = Paperclip::Thumbnail.new(closed_file, geometry: target_geometry_string, format: extension)
+      convert_options = '-strip -interlace Plane' if %w(jpg jpeg).include?(extension.to_s.downcase)
```

Comment:
strictly speaking, I feel `-strip ` option is unrelated to this PR and should not be included here